docs/getting-started/cli/init.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Initialising new Subsembly project
+---
+
+# subsembly init
+
+### Overview
+
+Initialises new **`Subsemly`** project to the specified path. Path to the new **`Subsembly`** project should be an empty or non-existing directory.
+
+### Syntax
+
+```text
+subsembly init [to]
+```
+
+#### Parameters
+
+* **`--to`** - \[optional\] initialisation directory for the new **`Subsembly`** project
```

Comment:
Maybe we can remove `[optional]` and describe it as:
"Optional parameter, used to determine the directory in which the Subsembly project will be initialised. By default the project is initialised in the current working directory"